android/titanium/src/java/org/appcelerator/titanium/view/TiUIView.java

```diff
@@ -895,21 +895,41 @@ public void propertyChanged(String key, Object oldValue, Object newValue, KrollP
 				ViewCompat.setElevation(nativeView, TiConvert.toFloat(newValue));
 			}
 		} else if (key.equals(TiC.PROPERTY_TRANSLATION_X)) {
-			if (nativeView != null) {
-				ViewCompat.setTranslationX(nativeView, TiConvert.toFloat(newValue));
+			if (getOuterView() != null) {
```

Comment:
Why exactly do we change the `nativeView` in various properties to 'getOuterView()', but leave properties like `elevation` to use the "old" way?